File: calc.java

public class calc{
    private int a,b;
    private String op;

    public calc(int left, String operation, int right){
	a = left;
	op = operation;
	b = right;
    }

    public void printValues(){
	System.out.print(a);
	System.out.print(op);
	System.out.print(b);
	System.out.println("");
    }

    public static void main(String[] args){
	calc calculator = new calc(Integer.parseInt(args[0]), args[1], Integer.parseInt(args[2]));
	calculator.printValues();
    }


}

File: calc.cpp
#include <iostream>
#include <cstdlib>
#include <string>

using namespace std;
class Calculator {
public:
  Calculator(int, string, int);
  int add(int a, int b);
  void printValues();
private:
  int a,b;
  string op;
};

Calculator::Calculator(int left, string operation, int right): a(left), b(right), op(operation){
}

void Calculator::printValues(){
  cout << a << op << b << endl;
}

int main(int argc, char* argv[]){
  Calculator calc( atoi(argv[1]), (string)argv[2], atoi(argv[3]) );
  calc.printValues();
  return 0;
}
